pkgs/servers/nosql/riak-cs/stanchion.nix

```diff
@@ -1,11 +1,12 @@
-{ stdenv, lib, fetchurl, unzip, erlang, git, wget, which, pam, coreutils, riak }:
+{ stdenv, lib, fetchurl, unzip, erlang, git, wget, which, pam, coreutils, riak 
+, Carbon ? null, Cocoa ? null }:
 
 stdenv.mkDerivation rec {
   name = "stanchion-2.1.1";
 
   buildInputs = [
     which unzip erlang pam git wget
```

Comment:
I generally prefer not to add null packages to `buildInputs` since they can't be used in strings and such.